### PR TITLE
Order fixing on custom questions

### LINKED ISSUE #120: Questions in reverse order
Questions appear to students in the reverse order provided in the form created by the instructors.

### PR DESCRIPTION
**Pull request by brendon-lind**
Reversed questions on the student submission view.

# Known issues to be addressed in a separate PR
Resolves #120 

# Checklist
- [x] Code review by me 
- [x] Hand tested by me 
- [x] I'm proud of my work
- [x] Code review by reviewer
- [x] Hand tested by reviewer
- [x] Ready to merge

### CODE REVIEW

File: static/riotjs/student/submit_homework.tag [hunk 1]
```diff
@@ -543,6 +543,7 @@
                 .done(function (data) {
                     self.definition = data
                     self.update()
+                    console.info("data", data)
```
Comment (gibsonbailey) [inline static/riotjs/student/submit_homework.tag]: Remove `console.info`

**Review by gibsonbailey — CHANGES_REQUESTED** [review]

File: static/riotjs/student/submit_homework.tag (hunk 1 above)
Comment (brendon-lind) [reply static/riotjs/student/submit_homework.tag]: My bad. Removed.

**Comment by gibsonbailey** [comment]
Can we ensure that questions appear in the same order on the homework definition form as the student submission form? This way, instructors can at least be aware of the order that will be presented to students, as it's usually important.

**Comment by brendon-lind** [comment]
It seems like it is working properly on this branch. Must have been something buggy on another branch before this fix.

**Pull request merged**